LICENSE_HEADER

```diff
@@ -1,6 +1,7 @@
 Jenkins plugin for Coding https://coding.net
 
-Copyright (C) 2016-2018 Shuanglei Tao <tsl0922@gmail.com>
+Copyright (c) 2016-2018 Shuanglei Tao <tsl0922@gmail.com>
+Copyright (c) 2016-present, Coding, Inc.
```

Comment:
`CODINT Tech`?

Reply:
Copy from https://github.com/facebook/react/blob/master/LICENSE

What is our company's name?